### ISSUE TITLE
Feature request - auto dismiss after N seconds

### ISSUE THREAD

**Issue by mrwaim** [djett41/ionic-content-banner#1]
Hi,

I'd like to dismiss the info automatically, after N seconds.

This works,

```
    var close = $ionicContentBanner.show({text:["Profile Saved"]});
    $timeout(function () {close();}, 1000);
```

But it depends on the "test hook" being exposed.

Is it popular enough to warrant an option in "function contentBanner (opts)"?

Thanks,
Ibrahim

**Comment by djett41** [comment]
your the first one to ask about it, but it seems like a reasonable feature to have!  I'll try to add it in when I get a chance thanks!

**Comment by rjlinden** [comment]
I would like this feature as well :)

**Comment by meneliksteiner** [comment]
Would be great if you could implement this ;)

**Comment by hughred22** [comment]
Upvote for it too. Auto dismiss will be super helpful.

**Comment by djett41** [comment]
@mrwaim @rjlinden @siam2nite @hughred22 Thanks guys!  This is available in the newest release 1.0.1.  Check the README for docs on autoClose

**Comment by rjlinden** [comment]
Thank you!! Highly appreciated. Will try it tonight. Keep up the good work!
